site/en/codelabs/silabs-openthread-hardware/index.lab.md

```diff
@@ -0,0 +1,438 @@
+
+### Setup and add FTD to our Thread network (out of band method)
+
+With the out-of-band method, we know all security information and add the node manually. In the Simplicity consoles add both FTDs to our network using the following commands in the order shown below.
+
+| Index | Command                                              | Command description                                          | Expected Response |
+| :---- | ---------------------------------------------------- | ------------------------------------------------------------ | ----------------- |
+| 1     | `dataset channel 20`                                 | Set the channel used by the OTBR                             | Done              |
+| 2     | `dataset masterkey 81ae2c2c17368d585dee71eaa8cf1e90` | Only the Master Key is required for a device to attach to a Thread network | Done              |
+| 3     | `dataset commit active`                              | Commit new dataset to the Active Operational Dataset         | Done              |
+| 4     | `ifconfig up`                                        | Enable Thread interface                                      | Done              |
+| 5     | `thread start`                                       | Enable and attach Thread protocol operation.                 | Done              |
+|       |                                                      | Wait 20 seconds while the device joins and configures itself. |                   |
+| 6     | `state`                                              | View network configuration.                                  | child<br/>Done    |
+
+> Note: Because of the self configuring nature of a Thread network and these being Full Thread devices, either or both the FTDs may eventually become  routers. This can be found out using the `state` command.
+
+### Communication between Thread devices
+
+We will use the `ping` command to check if devices can communicate with each other. To use the ping command we need the IPv6 addresses of the devices. These can be obtained using the `ipaddr` command.
+
+```console
+> ipaddr																	# IPv6 addresses of the OTBR
+fd5c:c6b:3a17:40b9:0:ff:fe00:fc00					# Leader Anycast Locator (ALOC)
+fd5c:c6b:3a17:40b9:0:ff:fe00:1800					# Routing Locator (RLOC)
+fd5c:c6b:3a17:40b9:84e2:bae8:bd5b:fa03		# Mesh-Local EID (ML-EID)
+fe80:0:0:0:c449:ca4a:101f:5d16						# Link-Local Address (LLA)
+```
+
+From both FTDs ping the OTBR using the OTBR's RLOC address.
+
+```console
+ping fd5c:c6b:3a17:40b9:0:ff:fe00:1800
+Done
+> 
+> 16 bytes from fd5c:c6b:3a17:40b9:0:ff:fe00:1800: icmp_seq=3 hlim=64 time=30ms
+16 bytes from fd5c:c6b:3a17:40b9:0:ff:fe00:1800: icmp_seq=3 hlim=64 time=52ms
+```
+
+ The response indicates the payload received and that the communication is successful. Repeat the process to ping FTDs from the OTBR.
+
+**You've created a Thread network!**
```

Comment:
Let's add a level 2 Congratulations! header here, and have that be the last step.

Further Reading can be a level 3 section under it.